<!DOCTYPE html>
<html lang="it">
  <head>
    <meta http-equiv="content-type" content="text/html; charset=windows-1252">
    <title>CASSMedChem links</title>
    <link rel="stylesheet" href="./styles.css">
  </head>
  <body>
    <h1>CASSMedChem links</h1>
    <h2>Sequence</h2>
    <p><b>Uniprot</b>: <a href="https://www.uniprot.org/">https://www.uniprot.org/</a></p>
    <ul>
      <li>The mission of UniProt is to provide the scientific community with a
        comprehensive, high-quality and freely accessible resource of protein
        sequence and functional information</li>
      <li>I use it to find protein sequences</li>
    </ul>
    <p><b>Blast</b>: <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a></p>
    <ul>
      <li>BLAST finds regions of similarity between biological sequences. The
        program compares nucleotide or protein sequences to sequence databases
        and calculates the statistical significance.</li>
      <li>Protein BLAST (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&amp;PAGE_TYPE=BlastSearch&amp;LINK_LOC=blasthome">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastp&amp;PAGE_TYPE=BlastSearch&amp;LINK_LOC=blasthome</a>)
        allows the search of similar sequence in PDB database</li>
    </ul>
    <h3>Protein Data Bank in Europe</h3>
    <ul>
      <li><b>Sequence similarity search</b>: <a href="https://www.ebi.ac.uk/pdbe-srv/PDBeXplore/sequence/">https://www.ebi.ac.uk/pdbe-srv/PDBeXplore/sequence/</a></li>
      <ul>
        <li>This PDB browser (PDBeXplore) is an interface that enables you to
          retrieve and analyse information on all the protein structures
          available in the Protein Data Bank (PDB) whose sequence display
          similarities to a sequence that you provide</li>
      </ul>
    </ul>
    <p><b>Smart</b>: <a href="http://smart.embl.de/">http://smart.embl.de/</a></p>
    <ul>
      <li>SMART (a Simple Modular Architecture Research Tool) allows the
        identification and annotation of genetically mobile domains and the
        analysis of domain architectures</li>
    </ul>
    <h2>Alignment</h2>
    <p><b>GUIDANCE 2: </b><a href="http://guidance.tau.ac.il/">http://guidance.tau.ac.il/</a></p>
    <ul>
      <li>Server for alignment confidence score<a href="http://guidance.tau.ac.il/"><br>
        </a></li>
    </ul>
    <p><b>MAFFT</b>: <a href="https://mafft.cbrc.jp/alignment/server/">https://mafft.cbrc.jp/alignment/server/<br>
      </a></p>
    <ul>
      <li>Multiple alignment program for amino acid or nucleotide sequences<a href="https://mafft.cbrc.jp/alignment/server/"><br>
        </a></li>
    </ul>
    <p><b>ProbCons</b>: <a href="http://probcons.stanford.edu/index.html">http://probcons.stanford.edu/index.html</a>
    </p>
    <ul>
      <li>PROBCONS is an efficient protein multiple sequence alignment program,
        which has demonstrated a statistically significant improvement in
        accuracy compared to several leading alignment tools</li>
      <li>The server is down but it is possible to download the software and
        compile it under Linux with some modifications (09/07/2020)</li>
      <ul>
        <li>size_t modified in std::size in all cc files</li>
        <li>#include &lt;string&gt; in #include &lt;cstring&gt; in cc files
          (even if it works changing only in ProjectPairwais.cc, Main.cc and
          CompareToRef.cc)</li>
      </ul>
    </ul>
    <p><b>PhyreRisk</b>: <a href="http://phyrerisk.bc.ic.ac.uk/home">http://phyrerisk.bc.ic.ac.uk/home</a></p>
    <ul>
      <li>PhyreRisk is a dynamic web application developed to enable the
        exploration and mapping of genetic variants onto experimental and
        predicted structures of proteins and protein complexes</li>
      <li>PhyreRisk integrates data from several public domain and in-house
        databases with information about diseases, genetic variation, biological
        pathways and protein structure and then facilitate the informed analysis
        by a wide community of users throughout the world.</li>
      <li>Alsin: <a href="http://phyrerisk.bc.ic.ac.uk/search?action=fresh-search&amp;searchTerm=Alsin">http://phyrerisk.bc.ic.ac.uk/search?action=fresh-search&amp;searchTerm=Alsin</a></li>
    </ul>
    <h2>ExPASy </h2>
    <h3>Alignment tools</h3>
    <ul>
      <li><b>ClustalW</b>: <a href="https://embnet.vital-it.ch/software/ClustalW.html">https://embnet.vital-it.ch/software/ClustalW.html</a></li>
    </ul>
    <h2>EMBL</h2>
    <ul>
      <li>Pfam: <a href="https://pfam.xfam.org/">https://pfam.xfam.org/</a></li>
      <ul>
        <li>The Pfam database is a large collection of protein families, each
          represented by multiple sequence alignments and hidden Markov models
          (HMMs)</li>
      </ul>
    </ul>
    <h2>Comparative modeling</h2>
    <p><b>Modeller:</b> <a href="https://salilab.org/modeller/">https://salilab.org/modeller/</a></p>
    <ul>
      <li>MODELLER is used for homology or comparative modeling of protein
        three-dimensional structures</li>
      <li>Tutorial basic: <a href="https://salilab.org/modeller/tutorial/basic.html">https://salilab.org/modeller/tutorial/basic.html</a></li>
      <li>Tutorial basic: <a href="http://vlab.amrita.edu/?sub=3&amp;brch=275&amp;sim=1481&amp;cnt=2">http://vlab.amrita.edu/?sub=3&amp;brch=275&amp;sim=1481&amp;cnt=2</a></li>
    </ul>
    <h2>Oligomerization/PPI modeling</h2>
    <p><b>HOMCOS: </b><a href="http://ipproo.protein.osaka-u.ac.jp/homcos/cgi-bin/index.cgi?LANG=en">http://ipproo.protein.osaka-u.ac.jp/homcos/cgi-bin/index.cgi?LANG=en</a></p>
    <ul>
      <li>HOMCOS(HOMology modeling of COmplex Structure) is a server for
        modeling complex 3D structures using 3D molecular similarities based on
        template complex 3D structures in PDB</li>
    </ul>
    <p><b>GalaxyWeb</b>: <a href="http://galaxy.seoklab.org/" title="GalaxyWeb">http://galaxy.seoklab.org/</a></p>
    <ul>
      <li>A web server for protein structure prediction, refinement, and related
        methods</li>
      <li>Tool used in Alsin modelisation <a href="http://galaxy.seoklab.org/cgi-bin/submit.cgi?type=DOCK">http://galaxy.seoklab.org/cgi-bin/submit.cgi?type=DOCK</a></li>
    </ul>
    <p><b>SPRING: </b><a href="https://zhanglab.ccmb.med.umich.edu/spring/">https://zhanglab.ccmb.med.umich.edu/spring/</a></p>
    <ul>
      <li>SPRING is a template-base algorithm for protein-protein complex
        structure prediction.</li>
    </ul>
    <p><b>ProtCID</b>: <a href="http://dunbrack2.fccc.edu/ProtCiD/default.aspx">http://dunbrack2.fccc.edu/ProtCiD/default.aspx</a></p>
    <ul>
      <li>ProtCID contains comprehensive, PDB-wide structural information on the
        interactions of proteins and individual protein domains with other
        molecules, including four types of interactions: chain interfaces, Pfam
        domain interfaces, Pfam-peptide interfaces and Pfam-ligand/nucleic acids
        interactions</li>
    </ul>
    <p><b>ZDock serve</b>r: <a href="http://zdock.umassmed.edu/">http://zdock.umassmed.edu/</a>
    </p>
    <ul>
      <li>ZDOCK and M-ZDOCK are Fast Fourier Transform based protein docking
        programs</li>
      <li>Tool used in Alsin modelisation</li>
    </ul>
    <h2>Software used in CASSMedChem lab</h2>
    <h2>Chimera</h2>
    <h3>Mutation</h3>
    <ul>
      <li><a href="https://www.youtube.com/watch?v=bcXMexN6hjY">Mutating a
          Residue in UCSF Chimera (Part 1)</a></li>
      <li><a href="https://www.youtube.com/watch?v=eJkrvr-xeXY">Mutating a
          Residue in USCF Chimera (Part 2)</a></li>
    </ul>
    <h3>Structure comparison</h3>
    <ul>
      <li><a href="https://www.youtube.com/watch?v=oThN3LG8LQU">UCSF Chimera:
          Structure Comparisons</a></li>
    </ul>
    <h2>GROMACS</h2>
    <ul>
      <li>Tutorials</li>
      <ul>
        <li>Lysozime in water:<a href="http://www.mdtutorials.com/gmx/lysozyme/08_MD.html">
            http://www.mdtutorials.com/gmx/lysozyme/08_MD.html</a></li>
        <ul>
          <li><span style="color: #dc3545;"></span>Basic tutorial to setup a
            simulation of a system containing a protein (lysozime) in a water
            box</li>
          <li>Tutorial used to setup standard MD simulations and Alsin dimer
            stability check</li>
        </ul>
      </ul>
    </ul>
    <p><br>
    </p>
    <p> </p>
    <ul>
    </ul>
    <p><br>
    </p>
    <p><br>
    </p>
    <p><br>
    </p>
  </body>
</html>
